variables.tf

```diff
@@ -39,6 +39,18 @@ variable "secrets" {
   description = "An object representing the secret to expose to the docker container"
 }
 
+variable "target_port" {
+  type        = number
+  default     = 80
+  description = "The target port"
+}
+
+variable "target_protocol" {
+  type        = string
+  default     = "HTTP"
+  description = "The target protocol"
+}
+
```

Comment:
It seems `secrets` was out of (alphabetical) order, and so now the new ones are also out of order.